examples/docker/docker-compose.yaml

```diff
@@ -33,9 +33,9 @@ services:
   # Sample web application which will generate Metrics and Traces data if the enable API is called
   # src - https://github.com/aws-observability/aws-otel-test-framework/tree/terraform/sample-apps/spark
   ot-sample-app:
-    image: aottestbed/aws-otel-collector-java-sample-app:0.6.0
+    image: public.ecr.aws/a3i2u7g3/aws-otel-test-framework-sample-apps-spark:1.4.1
```

Comment:
This needs to be changed from my public ecr repo to a public ecr repo owned by agents aws account.

Reply:
It seems this is using the docker hub link  https://hub.docker.com/layers/aottestbed/aws-otel-collector-java-sample-app/0.6.0/im[…]cd286de06a31a43da9765ef679fabe04989820e83a3a2?context=explore.

Reply:
Changed version to 0.10.0